### DISCUSSION TITLE
Amazon AWS S3 storage - exposed to public?

### DISCUSSION THREAD

**Discussion by jcat59** [monicahq/monica#5845] [General]
The instructions to link Monica to Amazon AWS storage, indicate that you should leave the settings to 'allow public access', which from what I understand literally means that anyone and everyone (in the world) can access it. Am I missing something, why would anyone want their Monica data to be publicly accessible?

Unless I'm missing something, can you please advise how to revert back to no longer use this storage? Would it just be changing the 'DEFAULT_FILESYSTEM' variable from S3 to ???

For now I shut down access on AWS so it wasn't public anymore.  As background - I followed the below instructions to set up external storage with Amazon AWS.
        https://github.com/monicahq/monica/blob/master/docs/installation/storage.md

**Comment by asbiin** [comment]
Thank you for your concern.
Some month ago I made a PR to fix that, so the public access to S3 is not needed anymore.
So to reassure you: **data on app.monicahq.com s3 bucket are not public anymore**

You can now set the privacy of the bucket to `private`, and it will work.
However, if for any reason you still want to use a public bucket, you can set `FILESYSTEM_DEFAULT_VISIBILITY` environment variable to `public` (default value is `private`).

The documentation is not up to date, I'll update it.

**Comment by jcat59** [comment]
Thanks for your reply, but it doesn’t work when blocking public access. I’m unable to update/add/change anything. see issue https://github.com/monicahq/monica/issues/5846#issuecomment-1008027422

**Comment by jcat59** [comment]
This is resolved now - after updating to the latest Monica sources
